packages/tangle/message.go

```diff
@@ -1232,3 +1299,18 @@ func (c *CachedMessageMetadata) Consume(consumer func(messageMetadata *MessageMe
 }
 
 // endregion ///////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// region Errors ///////////////////////////////////////////////////////////////////////////////////////////////////////
+var (
+	errNoStrongParents                    = errors.New("Missing strong messages in first parent block")
+	errBlocksNotOrderedByType             = errors.New("Blocks should be ordered in ascending order according to their type")
+	errBlockTypeIsUnknown                 = errors.Errorf("Block types must range from %d-%d", 0, numberOfBlockTypes-1)
+	errParentsCountMismatch               = errors.New("Number of parents in a message doesn't match parent count")
+	errParentsOutOfRange                  = errors.Errorf("A block must have at least %d-%d parents", MinParentsCount, MaxParentsCount)
+	errParentsNotLexicographicallyOrdered = errors.New("Messages within blocks must be lexicographically ordered")
+	errRepeatingBlockTypes                = errors.New("Block types within a message must not repeat")
+	errRepeatingReferencesInBlock         = errors.New("Duplicate parents in a message block")
+	errRepeatingMessagesAcrossBlocks      = errors.New("Different blocks have repeating messages")
```

Comment:
Error msgs need to be lowercased

Reply:
done

before I thought you just meant to make the vars unexported :-P
can we have our dog bark at this? I would expect this to be automated since making errors lower case is a golang thing..

Reply:
I've opened https://github.com/iotaledger/goshimmer/issues/1608 to track this. Can you please also make the error variables exported again? This would allow to match against these error cases from outside this package.

Reply:
We need this for integration tests maybe?

Because with our flat design it is anyhow pretty accessible

Reply:
I don't know where these errors would be examined right now but I could imagine that some situation might arise. For example integration tests as you mentioned :)